khelo/lib/ui/flow/tournament/detail/members/tournament_detail_members_screen.dart

```diff
@@ -0,0 +1,274 @@
+import '../../../matches/add_match/match_officials/search_user/search_user_screen.dart';
+
+class TournamentDetailMembersScreen extends ConsumerStatefulWidget {
+  final TournamentModel tournament;
+
+  const TournamentDetailMembersScreen({
+    super.key,
+    required this.tournament,
+  });
+
+  @override
+  ConsumerState<TournamentDetailMembersScreen> createState() =>
+      _TournamentDetailMembersScreenState();
+}
+
+class _TournamentDetailMembersScreenState
+    extends ConsumerState<TournamentDetailMembersScreen> {
+  late TournamentDetailMembersViewNotifier notifier;
+
+  @override
+  void initState() {
+    super.initState();
+    notifier = ref.read(tournamentDetailMembersStateProvider.notifier);
+    runPostFrame(() => notifier.setData(widget.tournament.members));
+  }
+
+  void _observeActionError(BuildContext context, WidgetRef ref) {
+    ref.listen(
+        tournamentDetailMembersStateProvider
+            .select((value) => value.actionError), (previous, next) {
+      if (next != null) {
+        showErrorSnackBar(context: context, error: next);
+      }
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    _observeActionError(context, ref);
+
+    final state = ref.watch(tournamentDetailMembersStateProvider);
+
+    return AppPage(
+      title: context.l10n.tournament_detail_members_title,
+      actions: state.currentUserId == widget.tournament.created_by
+          ? [
+              actionButton(
+                context,
+                icon: Icon(
+                  Icons.add,
+                  color: context.colorScheme.textPrimary,
+                ),
+                onPressed: () async {
+                  final user = await SearchUserBottomSheet.show<UserModel>(
+                    context,
+                    emptyScreenTitle: context
+                        .l10n.tournament_members_search_member_empty_title,
+                    emptyScreenDescription: context.l10n
+                        .tournament_members_search_member_empty_description,
+                  );
+
+                  if (user != null && mounted) {
+                    notifier.addTournamentMember(widget.tournament.id, user);
+                  }
+                },
+              )
+            ]
+          : null,
+      body: Builder(builder: (context) {
+        return _body(context, state);
+      }),
+    );
+  }
+
+  Widget _body(BuildContext context, TournamentDetailMembersState state) {
+    if (state.members.isEmpty) {
+      return EmptyScreen(
+        title: context.l10n.tournament_members_empty_title,
+        description: context.l10n.tournament_members_empty_description,
+        isShowButton: false,
+      );
+    }
+    final currantUserRole = state.members
+        .firstWhere(
+          (element) => element.id == state.currentUserId,
+        )
+        .role;
+
+    return ListView.separated(
+      padding: context.mediaQueryPadding + EdgeInsets.symmetric(vertical: 16),
```

Comment:
let's give this mediaQueryPadding to parent padding widget covering empty screens  also